content/definitions/czym-sa-dumb-smart-komponenty.md

```diff
@@ -0,0 +1,34 @@
+---
+question: Czym są dumb i smart komponenty?
+---
+
+`Smart komponenty` wykonują logiczne operacje, strzelają po dane, wykonują obliczenia, modyfikują dane, które później mogą
+przekazać do innych komponentów. Często `smart komponenty` nazywamy kontenerami, których zadaniem jest właśnie przejęcie całego mechanizmu strzelania po API, czy
+operacji na danych.
+
+
+`Dumb komponenty` natomiast są to komponenty, których rolą jest tylko i wyłącznie wyświetlenie/zaprezentowanie danych.
+
+Przykład `Smart` i `Dumb` komponentu.
+
+`Smart Component`
+```javascript
+export function SmartComponent() {
+    <strzał do Api> 
+    <przejęcie danych>
+
+    <przekazanie danych do komponentu DumbCompontent>
+}

```

Comment:
What do you think about using more valid javascript code in example? 
Smth simple, like: 
```javascript
function SmartComponent() {
   // strzał do API
    const data = fetch('https://api.jakas.strona').then(response => response.json())

   // przekazanie danych do DumbComponent
   <DubmComponent data={data} />
}
```

Especially if you are using React.js API in the dumb component example  😃

Reply:
Good hint !